drivers/soundwire/bus.c

```diff
@@ -747,6 +747,7 @@ static int sdw_program_device_num(struct sdw_bus *bus)
 
 		if (!found) {
 			/* TODO: Park this device in Group 13 */
+			sdw_slave_add(bus, &id, NULL);
```

Comment:
does that TODO still hold?

Reply:
@plbossart thanks for adding the comment :-) It makes me no smarter about what that "Group 13" is and why and when this device has to be parked there, but this isn't the purpose of this patch anyway :-) But what I'm wondering about wouldn't it be better to keep that comment together with the `dev_err()` call below? Either above or below your added chunk? But maybe I'm wrong and it really makes more sense the way it is now.